### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by Alex-1000**
- Sourcemod folder search now should work for Linux too
- Library imports have been edited to only import parts that are used in code
- Descriptions were written for functions
- Program asks for OS name only once now
- "message" and "message_quick" were merged into single function
- Some messages were merged together
- "message_yes_no" now has default answer option

### CODE REVIEW

**Comment by Alex-1000** [comment]
i forgot to sort libraries in some files, standard libraries are usually put above program's own

File: gui.py [hunk 1]
```diff
@@ -1,43 +1,57 @@
...
-def message_quick(msg):
-	print(msg)
-
-def message_yes_no(msg):
-	ans = None
-	while ans not in ("y", "n"):
-		ans = input(msg + " (Y/N)>").lower()
-	
-	if ans == "y":
+def message_yes_no(msg:str, default = true):
```
Comment (Technochips) [inline gui.py:13]: Not sure I like this when people could accidentally put in an invalid answer, then the program just chooses it for them.

**Review by Technochips — APPROVED** [review]
Merging message_quick with message is pretty good. Though we're planning on using the same functions to be able to actually show a GUI.

File: gui.py (hunk 1 above)
Comment (Alex-1000) [reply gui.py:13]: fair enough

**Review by Technochips — COMMENTED** [review]
I'd like to note that we're currently looking to use the registry.vdf file to get the sourcemods path on Linux.

**Comment by chloecormier** [comment]
LGTM, sorry for the wait. Even though some solutions are kind of dirty (like manually combing through possible paths for the sourcemods folder on Linux, which won't work for Flatpak, Snap, server installs using steamcmd...), we can improve on that after it's merged.

**Pull request merged**